test.js

```diff
@@ -1,17 +1,24 @@
 var test = require('tape')
-var nacl = require('tweetnacl')
+var base64 = require('base64-js')
 
-var session25519 = require('./')
+var session25519 = require('./index.js')
+// var session25519 = require('./dist/session25519.js')
 
-test('basic', function(t) {
-  session25519('user@example.com', 'secret', function(err, key) {
+test('deriving encrypt and sign keys', function (t) {
+  session25519('user@example.com', 'secret', function (err, key) {
     t.notOk(err, 'no error')
-    
+
     t.deepEqual(key, {
-      publicKey: nacl.util.decodeBase64('EHfJ2sJJt7LjamnuJul6TlCb5SkgAwJI1PP7q95jh04='),
-      secretKey: nacl.util.decodeBase64('p6FU22u0423AcS0AKY6I+XeBTMDzQLX57XOH9Xg28pw=')
-    }, 'correct key has been calculated')
-    
+      publicKey: base64.toByteArray('EHfJ2sJJt7LjamnuJul6TlCb5SkgAwJI1PP7q95jh04='),
```

Comment:
Same as before, yay! Therefore this appears backward-compatible with v1, and therefore theoretically compatible with miniLock, right @grempe?